suggestions/cloud_services.json

```diff
@@ -1 +1,30 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "com.owncloud.android",
+        "label": "ownCloud",
+        "source": "fg",
+        "reason": "Add an ownCloud server now and get... Your Cloud, Your Data, Your Way!\nhttps://owncloud.org/",
+        "repo": "https://github.com/owncloud/android"
+    },
+    {
+        "id": "com.infomaniak.drive",
+        "label": "Infomaniak kDrive",
+        "source": "fg",
+        "reason": "The swiss cloud service that doesn’t analyse or resell stored data.\nhttps://www.infomaniak.com/\nContains crash reports telemetry bits",
+        "repo": "https://github.com/Infomaniak/android-kDrive"
+    },
+    {
+        "id": "com.nextcloud.client",
+        "label": "Nextcloud",
+        "source": "fg",
+        "reason": "A safe home for all your data. Access & share your files, calendars, contacts, mail & more from any device, on your terms.\nhttps://nextcloud.com/",
```

Comment:
Reason is not for promotion, it's for specifying the reasons for including the app when the app's UI/UX deviates significantly from the reference bloatware. In this case, the UI/UX is mostly the same.